### PR TITLE
ARROW-11876: [Website] Update governance page

### PR DESCRIPTION
**Pull request by pitrou**
- Update chair to Wes from Jacques
- Reorganize YAML file to make it easier to edit and maintain
- Add a verification script to check the local list is up-to-date

### CODE REVIEW

**Comment by pitrou** [comment]
I realize some names are missing on our governance page:
* Deepak Majeti @majetideepak, [committer](https://home.apache.org/phonebook.html?uid=mdeepak)
* Ravindra Pindikura @pravindra, [committer](https://home.apache.org/phonebook.html?uid=ravindra)
* Todd Lipcon @toddlipcon, [PMC](https://home.apache.org/phonebook.html?uid=todd)
* Michael Stack @saintstack, [PMC](https://home.apache.org/phonebook.html?uid=stack)

To the people cc'ed above: I need your respective affiliations to add you to the list.

**Comment by pitrou** [comment]
Current output of verification script:
```
$ ./scripts/check_committers.py 
Missing PMCs in local list: ['stack', 'todd']
Missing committers in local list: ['alamb', 'mdeepak', 'ravindra']
```

**Comment by wesm** [comment]
Todd and Stack both indicated that they wanted to go emeritus from the PMC in August 2019

**Comment by pitrou** [comment]
Apparently that wasn't done at the ASF level?

File: _data/committers.yml [hunk 1]
```diff
@@ -15,203 +15,206 @@
...
-  alias: adeneche
-  affiliation: Salesforce
-- name: Wes McKinney
+  alias: emkornfield
+  affiliation: Google
+- name: Neal Richardson
   role: PMC
-  alias: wesm
+  alias: npr
   affiliation: Ursa Labs / RStudio
```
Comment (nealrichardson) [inline _data/committers.yml:91]:
```suggestion
  affiliation: Ursa Computing
```

File: _data/committers.yml [hunk 2]
```diff
@@ -15,203 +15,206 @@
...
-- name: James Taylor
+  alias: apitrou
+  affiliation: Independent / Ursa Labs
+- name: François Saint-Jacques
   role: PMC
-  alias: jamestaylor
-  affiliation: Salesforce
-- name: Julien Le Dem
+  alias: fsaintjacques
+  affiliation: Ursa Labs / RStudio
```
Comment (nealrichardson) [inline _data/committers.yml:43]: @fsaintjacques what's your company called again?

File: _data/committers.yml [hunk 3]
```diff
@@ -15,203 +15,206 @@
...
 #
-- name: Jacques Nadeau
+
+- name: Wes McKinney
   role: VP
-  alias: jacques
-  affiliation: Dremio
-- name: Ted Dunning
+  alias: wesm
+  affiliation: Ursa Labs / RStudio
```
Comment (nealrichardson) [inline _data/committers.yml:22]:
```suggestion
  affiliation: Ursa Computing
```

File: _data/committers.yml [hunk 4]
```diff
@@ -15,203 +15,206 @@
...
+  alias: julien
+  affiliation: Datakin
+- name: Kouhei Sutou
+  role: PMC
+  alias: kou
+  affiliation: ClearCode
+- name: Krisztián Szűcs
+  role: PMC
+  alias: kszucs
+  affiliation: Ursa Labs
```
Comment (nealrichardson) [inline _data/committers.yml:79]:
```suggestion
  affiliation: Ursa Computing
```

File: _data/committers.yml [hunk 5]
```diff
@@ -15,203 +15,206 @@
...
-- name: Reynold Xin
+  alias: agrove
+  affiliation: NVIDIA
+- name: Antoine Pitrou
   role: PMC
-  alias: rxin
-  affiliation: Databricks
-- name: James Taylor
+  alias: apitrou
+  affiliation: Independent / Ursa Labs
```
Comment (nealrichardson) [inline _data/committers.yml:39]:
```suggestion
  affiliation: Independent / Ursa Computing
```

File: _data/committers.yml [hunk 6]
```diff
@@ -15,203 +15,206 @@
...
+- name: David Alves
   role: Committer
-  alias: praveenbingo
-  affiliation: Dremio
+  alias: dralves
+  affiliation: CortexXus
 - name: David Li
   role: Committer
   alias: lidavidm
   affiliation: Two Sigma
```
Comment (nealrichardson) [inline _data/committers.yml:156]:
```suggestion
  affiliation: Ursa Computing
```

File: _data/committers.yml [hunk 7]
```diff
@@ -15,203 +15,206 @@
...
+  alias: tianchen
+  affiliation: Alibaba
+- name: Jorge Cardoso Leitao
+  role: Committer
+  alias: jorgecarleitao
+  affiliation: Munin Data ApS
 - name: Joris van den Bossche
   role: Committer
   alias: jorisvandenbossche
   affiliation: Ursa Labs
```
Comment (nealrichardson) [inline _data/committers.yml:176]: Ursa Computing

File: _data/committers.yml [hunk 8]
```diff
@@ -15,203 +15,206 @@
...
+  alias: uwe
+  affiliation: Quantco
+
+- name: Ben Kietzman
   role: Committer
-  alias: shiro
-  affiliation: Red Data Tools
-- name: Romain Francois
+  alias: bkietz
+  affiliation: Ursa Labs / RStudio
```
Comment (nealrichardson) [inline _data/committers.yml:136]:
```suggestion
  affiliation: Ursa Computing
```

File: _data/committers.yml [hunk 9]
```diff
@@ -15,203 +15,206 @@
 #
 # Database of Apache Arrow committers and PMC
 #
-- name: Jacques Nadeau
+
```
Comment (nealrichardson) [inline _data/committers.yml:18]: Add a comment pointing to the script you added for verification?

**Review by nealrichardson — CHANGES_REQUESTED** [review]
Some affiliations need updating

File: _data/committers.yml [hunk 10]
```diff
@@ -15,203 +15,206 @@
...
+  alias: romainfrancois
+  affiliation: RStudio
 - name: Yibo Cai
   role: Committer
   alias: yibocai
   affiliation: Arm
+- name: Yosuke Shiro
+  role: Committer
+  alias: shiro
+  affiliation: Red Data Tools
```
Comment (alamb) [inline _data/committers.yml:220]:
```suggestion
- name: Yosuke Shiro
  role: Committer
  alias: shiro
  affiliation: Red Data Tools
- name: Andrew Lamb
  role: Committer
  alias: alamb
  affiliation: InfluxData  
```

**Review by alamb — COMMENTED** [review]
Thank you @pitrou

File: _data/committers.yml (hunk 2 above)
Comment (fsaintjacques) [reply _data/committers.yml:43]: :wave:, it's called Optable.

**Comment by pitrou** [comment]
Since other people didn't chime in, I'm simply going to merge this.

**Pull request closed without merging**